src/Schema/Plugin/SortPlugin.php

```diff
@@ -86,17 +87,67 @@ public function apply(Field $field, Schema $schema, array $config = []): void
     public static function sort(array $context): Closure
     {
         $fieldName = $context['fieldName'];
-        return function (?Sortable $list, array $args) use ($fieldName) {
+        return function (?Sortable $list, array $args, array $context, ResolveInfo $info) use ($fieldName) {
             if ($list === null) {
                 return null;
             }
```

Comment:
Return early if there's no sort argument for this query.
```suggestion
            }

            if (!isset($args[$fieldName])) {
                return $list;
            }
```

Reply:
Updated